chef/cookbooks/ceph/recipes/conf.rb

```diff
@@ -6,7 +6,14 @@
 
 mon_init = []
 mon_nodes.each do |monitor|
-    mon_init << monitor.name.split(".")[0]
+  mon_name = monitor.name.split(".")[0]
```

Comment:
Here we are creating node_name for remote monitors in the cluster, function get_node_name operates on local node object. Call here `monitor.get_node_name` will throw 
```
ArgumentError
-------------
Attribute node is not defined!
```